src/main/resources/variables/entities.yml

```diff
@@ -40,6 +40,13 @@
   pattern: |
     [lemma = /plant|sow|seed/  & tag = /^VB/] 
 
+- name: variable-area
+  priority: ${rulepriority}
+  label: Area
+  type: token
+  pattern: |
+    ([lemma=/cultivate|plant|sown/])? [lemma=/area/] ([lemma=/sown/])?
```

Comment:
I don't know anything about this, but it seems strange that "sown" would be a lemma.  Wouldn't it be "sow"?

Reply:
Yes, absolutely should be sow. Thank you!